src/main/scala/scalaz/analytics/AnalyticsModule.scala

```diff
@@ -138,6 +149,13 @@ trait AnalyticsModule {
   implicit def double[A](v: scala.Double): A =>: Double
   implicit def decimal[A](v: scala.BigDecimal): A =>: BigDecimal
   implicit def string[A](v: scala.Predef.String): A =>: String
+  implicit def boolean[A](v: scala.Boolean): A =>: String
+  implicit def byte[A](v: scala.Byte): A =>: Byte
+  // TODO;  Why Null? Or we need to support a type B that can be Null (implicit ev: scala.Null <:< B) ?
```

Comment:
No I think we want `Null` directly as you've implemented it. Safe to remove this comment :)

We explicitly want to disallow the quirk of scala that `Int` can be set to null for example. Most of this stuff will be operating on unboxed primitives in our interpreter which means a `null` value would be somewhat nonsensical (`Null` is not a subtype of `AnyVal`).